setup.cfg

```diff
@@ -0,0 +1,49 @@
+[metadata]
+name = disk_objectstore
+version = attr: disk_objectstore.__version__
+description = An implementation of an efficient object store writing directly into a disk folder
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = http://github.com/aiidateam/disk-objectstore
+author = Giovanni Pizzi
+license = MIT
+license_file = LICENSE.txt
+classifiers =
+    Development Status :: 4 - Beta
+    License :: OSI Approved :: MIT License
+    Operating System :: MacOS :: MacOS X
+    Operating System :: Microsoft :: Windows
+    Operating System :: POSIX :: Linux
+    Programming Language :: Python :: 3
+    Programming Language :: Python :: 3.6
+    Programming Language :: Python :: 3.7
+    Programming Language :: Python :: 3.8
+    Programming Language :: Python :: 3.9
+    Programming Language :: Python :: Implementation :: CPython
+    Topic :: Software Development :: Libraries :: Python Modules
+keywords = object store, repository, file store
+
+[options]
+packages = find:
+install_requires =
+    sqlalchemy<1.4
+python_requires = ~=3.7
+include_package_data = True
+
+[options.packages.find]
+exclude =
+    test*
+
+[options.extras_require]
+dev =
+    coverage
+    pre-commit
+    pytest
+    pytest-benchmark
+    pytest-cov
+examples =
+    click
+    memory-profiler
+    profilehooks
+    psutil
+    pywin32;platform_system == "Windows"
```

Comment:
setup.json is a bespoke aiida solution, not supported by any PEP. The only reason we use it, is for compatibility with the aiida-registry (which eventually we should fix: https://github.com/aiidateam/aiida-registry/issues/132, but this will be easier when everything moves to `pyproject.toml`)